internal/pkg/plugin/argocd/update.go

```diff
@@ -10,12 +10,16 @@ import (
 )
 
 func Update(options map[string]interface{}) (map[string]interface{}, error) {
-	// 1. decode options
+	// 1. decode options and fill default options if miss
 	var opts Options
 	if err := mapstructure.Decode(options, &opts); err != nil {
 		return nil, err
 	}
 
+	if err := defaultMissedOptions(&opts); err != nil {
```

Comment:
CI prompted a compilation error, it seems you forgot to change here:

<img width="730" alt="image" src="https://user-images.githubusercontent.com/36830265/179345231-ffdd50fd-4359-428b-a531-7db85794d8b6.png">

Reply:
my bad, i have committed a fix, do i need to rebase and squash the commit message?